README.md

```diff
@@ -35,6 +35,12 @@ A list of every a11y concern Checka11y.css will check for and highlight:
 
 - Missing `alt` attributes on images
 - Missing `title` on `<iframe>`
+<<<<<<< HEAD
+- Checks `<li>` is the **only** child of `<ol>` and `<ul>`
```

Comment:
I'm not sure this file should have been changed, please double check it is up to date with `master`.